sphinxcontrib/confluencebuilder/publisher.py

```diff
@@ -848,6 +848,7 @@ def _update_page(self, page, page_name, data, parent_id=None):
         update_page['id'] = page['id']
         update_page['version'] = {
             'number': last_version + 1,
+            'message': self.config.confluence_version_comment,
```

Comment:
It works.
The value `None` produces no error, and the page is uploaded with an empty comment.
![image](https://user-images.githubusercontent.com/19349405/157398101-3993b541-912e-4e27-9f4b-deab01dccda0.png)

So, I don't think that we need mentioned tweak. Is that ok with you @jdknight?

Reply:
It could be ~ I'd be curious to see how the oldest version of Requests we support handles it (does a `None` translate to not sending it in some, empty in others, etc.), and also checking against the older LTS version of Confluence supported handles it as well (does all Confluence instances accept an empty comment field if set). I assume it will be fine, but I seen the oddest corner cases sometimes -- I can spawn up an instance and sanity check this soon (tommorrow?).

Reply:
Looks good on the older instances of Confluence; going to continue with the merge.